packages/grid/_modules_/grid/constants/localeTextConstants.ts

```diff
@@ -46,6 +46,7 @@ export const DEFAULT_LOCALE_TEXT: LocaleText = {
   filterOperatorEndsWith: 'ends with',
   filterOperatorIs: 'is',
   filterOperatorNot: 'is not',
+  filterOperatorAfter: 'is after',
```

Comment:
Should be into its own PR

```suggestion
```

The next release will likely be published before we merge this PR. If we don't, we store value, value we could have provided to the users sooner and in smaller iterations. Here, should it throw when a translation is missing? Or do we want to change the tradeoff with a warning?

Reply:
Yes but this issue is blocking me from testing the grid, and testing the feature I'm currently building.
I thought Danail fixed it, I just need to rebase

Reply:
Sounds great